test/evp_libctx_test.c

```diff
@@ -468,11 +468,8 @@ int setup_tests(void)
         return 0;
 
     libctx = OPENSSL_CTX_new();
-
-    if (!TEST_ptr(libctx))
-        return 0;
-
-    if (config_file != NULL) {
+    if (!TEST_ptr(libctx)
+        || config_file != NULL) {
```

Comment:
The condition seems wrong to me. You're calling OPENSSL_CTX_load_config if the OPENSSL_CTX_new fails? Why?

Reply:
Yep it is wrong. Changing now. Thanks.